--- WORKSPACE 2016-07-05 08:25:21.814380921 +0000
+++ WORKSPACE.20160705    2016-07-05 08:28:42.866172703 +0000
@@ -1,18 +1,18 @@
 workspace(name = "org_tensorflow")
 
 # Uncomment and update the paths in these entries to build the Android demo.
-#android_sdk_repository(
-#    name = "androidsdk",
-#    api_level = 23,
-#    build_tools_version = "23.0.1",
-#    # Replace with path to Android SDK on your system
-#    path = "<PATH_TO_SDK>",
-#)
-#
-#android_ndk_repository(
-#    name="androidndk",
-#    path="<PATH_TO_NDK>",
-#    api_level=21)
+android_sdk_repository(
+    name = "androidsdk",
+    api_level = 23,
+    build_tools_version = "24.0.0",
+    # Replace with path to Android SDK on your system
+    path = "/opt/android-sdk-linux/",
+)
+
+android_ndk_repository(
+    name="androidndk",
+    path="/opt/android-ndk-r12b/",
+    api_level=21)
 
 # Please add all new TensorFlow dependencies in workspace.bzl.
 load("//tensorflow:workspace.bzl", "tf_workspace")
